### DISCUSSION TITLE
Feishu alert issue.

### DISCUSSION THREAD

**Discussion by JoneQiang21** [apache/dolphinscheduler#11039] [Q&A]
Please give me some advice about this issue, thx.
@Tandoy  @lenboo thx

![image](https://user-images.githubusercontent.com/108962672/179670070-a9d04e17-45be-4ae0-9ba5-2f1bfad085c7.png)

**Comment by Tandoy** [comment]
provide your config file,there may be a problem with the port configuration

**Reply by JoneQiang21** [reply]
> provide your config file,there may be a problem with the port configuration

I didnot config anything and I didnot find anything about feishu alert on Document. Could you give me some advice about it. thx  @Tandoy

**Reply by JoneQiang21** [reply]
Just config it on dolphin's web UI

**Comment by Tandoy** [comment]
You may need to configure it in alter.properties.Refer to the following parameter classes

https://github.com/apache/dolphinscheduler/blob/2.0.5-release/dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-feishu/src/main/java/org/apache/dolphinscheduler/plugin/alert/feishu/FeiShuParamsConstants.java

**Reply by JoneQiang21** [reply]
> You may need to configure it in alter.properties.Refer to the following parameter classes
> 
> https://github.com/apache/dolphinscheduler/blob/2.0.5-release/dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-feishu/src/main/java/org/apache/dolphinscheduler/plugin/alert/feishu/FeiShuParamsConstants.java


./conf/quartz.properties
./conf/python-gateway.properties
./conf/i18n/messages_en_US.properties
./conf/i18n/messages_zh_CN.properties
./conf/i18n/messages.properties
./conf/common.properties
./conf/registry.properties
./conf/worker.properties
./conf/application-api.properties
./conf/master.properties

I did not see there is alert.properties file ,is it normal?

**Reply by JoneQiang21** [reply]
> 

@Tandoy

**Reply by Tandoy** [reply]
Sorry, I have version 1.3.6. After installation, there is alert Properties this file may not exist after subsequent upgrades. You can use F12 when creating an alarm instance on the page to check whether the port in the parameter plugininstanceparams in the request meets the requirements of the source code.

https://github.com/apache/dolphinscheduler/blob/2.0.5-release/dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-feishu/src/main/java/org/apache/dolphinscheduler/plugin/alert/feishu/FeiShuSender.java#L59
